examples/with-supertokens/package.json

```diff
@@ -10,9 +10,9 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "supertokens-auth-react": "^0.18.0",
-    "supertokens-node": "^8.1.0"
+    "supertokens-node": "^9.0.0"
   },
   "devDependencies": {
```

Comment:
Actually, let's get rid of `devDependencies` all together, as eslint isn't used here.